xil_cloud_telematics/carma.config.js

```diff
@@ -41,34 +41,16 @@ CarmaJS.registerNamespace("CarmaJS.Config");
 
 CarmaJS.Config = (function () {
         //Private variables
-        var ip = '127.0.0.1'; //'192.168.88.10'; 192.168.32.146;
-        var refresh_interval = 30; //30 seconds
-        var ros_connect_wait = 5000; //5 miliseconds to wait for platform to launch and ros to connect.
-        var ros_connect_retry = 24; //# of times to wait. Total 2 minutes
-        var speed_limit = 30; //Default Speed Limit
```

Comment:
Is there a reason for removing these? They're present in the physical vehicle configs, and while I'm not totally sure what they do, I don't think we intentionally removed these in sim?

Reply:
Looks like the definitions in these files relate to carma-web-ui requests which is not being used with this config